packages/node/src/Client.ts

```diff
@@ -74,6 +75,22 @@ class Client {
     return this.makeHttpRequest('identify', params)
   }
 
+  generateIdentityToken(uid: string): string {
```

Comment:
Will revisit this in cases when ids are integer. Will have to accomodate this on the go side as well